docs/guides/hooks/slack-notification/index.md

```diff
@@ -346,34 +408,92 @@ spec:
                 }
               ]
           }
-  retentionPolicy:
-    name: 'keep-last-5'
-    keepLast: 5
-    prune: true
+  executor:
+    type: Operator
+```
+
+Let’s create the above `HookTemplate`,
+
+```bash
+$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/hooks/slack-notification/examples/hooktemplate_failed.yaml
+hooktemplate.core.kubestash.com/slack-hook configured
+```
+
+We can also use the `executionPolicy` to send notification only for the failed backups. Check the following BackupConfiguration:
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: BackupConfiguration
+metadata:
+  name: sample-backup
+  namespace: demo
+spec:
+  target:
+    apiGroup: apps
+    kind: DaemonSet
+    name: ks-demo
+    namespace: demo
+  backends:
+    - name: gcs-backend
+      storageRef:
+        namespace: demo
+        name: gcs-storage
+      retentionPolicy:
+        name: demo-retention
+        namespace: demo
+  sessions:
+    - name: frequent-backup
+      sessionHistoryLimit: 3
+      scheduler:
+        schedule: "*/5 * * * *"
+        jobTemplate:
+          backoffLimit: 1
+      hooks:
+        postBackup:
+          - name: slack-notification
+            hookTemplate:
+              name: slack-hook
+              namespace: demo
+            maxRetry: 3
+            timeout: 30s
+            executionPolicy: OnFailure
+      repositories:
+        - name: daemon-repo
+          backend: gcs-backend
+          directory: /demo/daemon
+          encryptionSecret:
+            name: encrypt-secret
+            namespace: demo
+      addon:
+        name: workload-addon
+        tasks:
+          - name: logical-backup
+            params:
+              paths: /wrong/data
+      retryConfig:
+        maxRetry: 2
+        delay: 1m
 ```
 
 Here, we have provided an invalid path in the `paths` field of the `target` section. This will force the backup to fail.
 
-Notice that, this time we have specified the `executionPolicy` field to `OnFailure`. This will tell Stash to send the notification only if the backup fail. In the message body, we have included information about target and reason of failure.
+Notice that, this time we have specified the `executionPolicy` field to `OnFailure`. This will tell KubeStash to send the notification only if the backup fail. In the message body, we have included information about target and reason of failure.
```

Comment:
```suggestion
Notice that, this time we have specified the `executionPolicy` field to `OnFailure`. This will tell KubeStash to send the notification only if the backup fails. In the message body, we have included information about target and reason of the failure.
```